lib/rbovirt.rb

```diff
@@ -97,9 +97,12 @@ def floppy_hook?
     end
 
     private
+
     def search_url opts
-      search = opts[:search] || ("datacenter=%s" % current_datacenter.name)
-      "?search=%s" % CGI.escape(search)
+      search = opts[:search] || ("datacenter=\"%s\"" % current_datacenter.name)
+      search += " page #{opts[:page]}" if opts[:page]
```

Comment:
I agree, if I understand correctly the user provided search should be added to the datacenter filter instead of replacing it?